### PR TITLE
API Docs

### PR DESCRIPTION
**Pull request by peteclark-ft**
Dredd passes, except for the /__gtg endpoint, which needs a working ES cluster (more than one ES node) to pass.

### CODE REVIEW

File: api/api.yml [hunk 1]
```diff
@@ -0,0 +1,202 @@
+swagger: "2.0"
+
+info:
+   title: Concept Search API
+   description: A microservice that provides endpoints to query for UPP V2 Concepts
+   version: 0.0.1
+   contact:
+      name: Universal Publishing
+      email: Universal.Publishing@ft.com
```
Comment (khatton-ft) [inline api/api.yml]: I'd stick to lower case for the email address.

File: api/api.yml [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+basePath: /
+
+paths:
+   /concepts:
+      get:
+         summary: Concept Query
+         description: Get all Concepts by type or search for concepts using a query.
+         tags:
+            - Public API
+         parameters:
```
Comment (khatton-ft) [inline api/api.yml]: The `mode` parameter is missing here.

File: api/api.yml [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+               description: The query text to use to find concepts.
+               type: string
+               required: false
+         responses:
+            200:
+               description: Returns concepts based on the provided query parameters.
+               examples:
+                  application/json:
+                     concepts:
+                        -  id: http://api.ft.com/things/61d707b5-6fab-3541-b017-49b72de80772
```
Comment (khatton-ft) [inline api/api.yml]: `id` format should be `http://www.ft.com/thing/...`

File: api/api.yml (hunk 2 above)
Comment (peteclark-ft) [inline api/api.yml]: It wasn't in master at the time of writing, so I decided not to add it - I can add it if that's been merged now?

Comment (peteclark-ft) [inline api/api.yml]: Added now.

**Pull request merged**